azure-pipelines.yaml

```diff
@@ -91,6 +91,37 @@ parameters:
       - plan
       - apply
 
+  # Used for building MI infrastructure across the environments
+  - name: environment_components
```

Comment:
I would just call it enviroments or service connections, could be used on multiple components if needs be.

Reply:
Agreed, will rename